docs/reference/how-to/disk-usage.asciidoc

```diff
@@ -158,3 +158,27 @@ on disk usage. In particular, integers should be stored using an integer type
 stored in a `scaled_float` if appropriate or in the smallest type that fits the
 use-case: using `float` over `double`, or `half_float` over `float` will help
 save storage.
+
+[float]
+=== Use index sorting to colocate similar documents
+
+Elasticsearch compresses multiple documents at once in order to improve the
+overall compression ratio. For instance it is very common that documents share
+the same field names, and quite common that they share some field values,
+especially on fields that have a low cardinality or a zipfian distribution.
+
+Documents that are compressed together are documents that are colocated in the
```

Comment:
I'd twist this around to something like "By default documents are compressed together in the order that they are added to the index. If you enabled index sorting then instead they are compressed in sorted order. Sorting documents with similar structure, fields, and values together *should* improve the compression ratio." Or something like that. It feels more active that way. I dunno.